server/src/routes/auth.js

```diff
@@ -0,0 +1,115 @@
+import cors from 'cors';
+import bodyParser from 'body-parser';
+import jwt from 'jsonwebtoken';
+import bcrypt from 'bcrypt';
+import cookieParser from 'cookie-parser';
+
+import database from '../database/db';
+import config from '../../config';
+import { SESSION_COOKIE } from '../app';
+
+export default function(router) {
+  router.use(cors({
+    origin: ['http://localhost:3000'], // origin: ['http://localhost:8000'], //can change to 8000 on deploy
+    credentials: true
+  }));
+  router.use(bodyParser.json());
+  router.use(cookieParser());
+
+  router.post('/login', (req, res) => {
+    const { email, password } = req.body.login;
+    // select username and password from database
+    const querySQL = `select
+                        user_id,
+                        password
+                      from
+                        users
+                      where
+                        email like '${email}';`;
+
+     database.query(querySQL, [])
+      .then((response) => {
+
+        // check for existing user data
+        if (response.rows.length > 0) {
+          // decrypt password
+          const decryptedPassword = bcrypt.compareSync(password, response.rows[0].password);
+
+          if (decryptedPassword) {
+            // create session cookie
+            const sessionUser = { email: email };
+            const JWT = jwt.sign(sessionUser, config.get('APP_SECRET'));
+
+            res.status(200).cookie(SESSION_COOKIE, JWT, {
+              // secure: config.get('HTTPS'), // for production
+              secure: false,
+              maxAge: 7200000,
+              httpOnly: true,
+            }).json({ response: true, userId: response.rows[0].user_id })
+          } else {
+            res.status(200).json({ response: false });
```

Comment:
Should we be returning a `200` if the decryption fails? Shouldn't that be a `401`?